examples/viewer_fps_label.py

```diff
@@ -0,0 +1,17 @@
+"""
+Display a 3D volume and the fps label
```

Comment:
Docstrings are sentences, so it is generally recommended to use a period at the end.